### ISSUE TITLE
How to solve the problem of downloading large images leads to high memory and flashback

### ISSUE THREAD

**Issue by XavierLost** [SDWebImage/SDWebImage#2870]
### New Issue Checklist

* [ ] I have read and understood the [CONTRIBUTING guide](https://github.com/rs/SDWebImage/blob/master/.github/CONTRIBUTING.md)
* [ ] I have read the [Documentation](http://cocoadocs.org/docsets/SDWebImage/)
* [ ] I have searched for a similar issue in the [project](https://github.com/rs/SDWebImage/issues) and found none

### Issue Info

 Info                    | Value                               |
-------------------------|-------------------------------------|
 Platform Name           | e.g. ios / macos / tvos / watchos
 Platform Version        | e.g. 12.0 / 10.14.0 / 12.0 / 5.0
 SDWebImage Version      | e.g. 5.0.0 / 4.4.0
 Integration Method      | e.g. carthage / cocoapods / manually
 Xcode Version           | e.g. Xcode 11 / Xcode 10
 Repro rate              | e.g. all the time (100%) / sometimes x% / only once
 Repro with our demo prj | e.g. does it happen with our demo project?
 Demo project link       | e.g. link to a demo project that highlights the issue

### Issue Description and Steps
Is there a way to solve the problem of downloading large images, resulting in high memory and flashback?

Please fill in the detailed description of the issue (full output of any stack trace, compiler error, ...) and the steps to reproduce the issue.

**Comment by dreampiggy** [comment]
Duplicated for #1544
See: [Wiki - Optimization for large images](https://github.com/SDWebImage/SDWebImage/wiki/Common-Problems#optimization-for-large-images)

**Comment by XavierLost** [comment]
SDWebImageManager.sharedManager.optionsProcessor = [SDWebImageOptionsProcessor optionsProcessorWithBlock： ^ SDWebImageOptionsResult * _Nullable（NSURL * _Nullable url，SDWebImageOptions options，SDWebImageContext * _Nullable context）{
      //禁用全局强制解码，可能会降低帧速率
     选项| = SDWebImageAvoidDecodeImage;
     返回 [[SDWebImageOptionsResult alloc ] initWithOptions： options context： context];
}];
